src/App.tsx

```diff
@@ -1,24 +1,198 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  createTodo,
+  deleteTodo,
+  editTodo,
+  getTodos,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './types/TodoFilter';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notifications } from './components/Notifications';
 
-const USER_ID = 0;
+const USER_ID = 11399;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [errors, setErrors] = useState({
+    load: false,
+    delete: false,
+    empty: false,
+    add: false,
+    edit: false,
+  });
+  const [currentFilter, setCurrentFilter] = useState(TodoFilter.All);
+  const [loadingId, setLoadingId] = useState<number[]>([]);
```

Comment:
```suggestion
  const [loadingTodoIds, setLoadingTodoIds] = useState<number[]>([]);
```
array - use plural form